src/index.jsx

```diff
@@ -29,6 +30,7 @@ const crashReporter = () => next => action => {
 
 const store = createStore(
   combineReducers({ experiments, dates }),
+  new Map(),
```

Comment:
What does this do?

Reply:
This was Rehan's tip! Pass into the second argument a new Map so that when the store is created, it's padded with an initial structure (otherwise it'll be undefined). 

This came about when I was trying to call get("experiments")....get("something") in my selector, and it kept on crashing. I was calling get() on undefined since the fetch call to the API was still in progress #iLoveAsynchronism

Reply:
Interesting 🤔

Reply:
Might be useful for everyone: https://redux.js.org/api/createstore